examples/upsert-resource/simple/.expected/diff.patch

```diff
@@ -0,0 +1,50 @@
+diff --git a/fn-config.yaml b/fn-config.yaml
+deleted file mode 100644
+index da512ae..0000000
+--- a/fn-config.yaml
++++ /dev/null
+@@ -1,12 +0,0 @@
+-apiVersion: v1
+-kind: Service
+-metadata:
+-  name: myService
+-  namespace: mySpace
+-  annotations:
+-    config.k8s.io/function: |
+-      container:
+-        image: gcr.io/kpt-fn/upsert-resource:unstable
+-spec:
+-  selector:
+-    app: bar
+diff --git a/resources.yaml b/resources.yaml
+index 50886bd..debc420 100644
+--- a/resources.yaml
++++ b/resources.yaml
+@@ -3,9 +3,26 @@ kind: Service
+ metadata:
+   name: myService
+   namespace: mySpace
++  annotations:
++    config.k8s.io/function: |
++      container:
++        image: gcr.io/kpt-fn/upsert-resource:unstable
```

Comment:
This shouldn't happen (for 0.38). It will not happen for 1.0 by virtue of the annotations going away.

Reply:
Updated.

Reply:
Not updated?

The issue is that all other functions work for both 0.38 and 1.0. We are not breaking backwards compatability for functions in 1.0. This would be the first function where it may not work well with 0.38.
 
Why can't you just look for this annotation and delete it?

Reply:
kpt v0.X.Y treats function configs as resources too if they are present in the package. So functions have to filter them. Did a refactor. This is compatible with both v0.X.Y and v1 now. PTAL.